test/built-ins/Simd/Simd.negate.js

```diff
@@ -0,0 +1,32 @@
+//  Ecma TC39 modifications to this file are subject to the following:
+//  Copyright (C) 2015 [Deepti Gandluri]. All rights reserved.
+//  This code is governed by the BSD license found in the LICENSE file.
+//
+//  The initial version of this file as brought to Ecma TC39 includes the
+//  following notice:
+//  This software is provided 'as-is', without any express or implied
+//  warranty.  In no event will the authors be held liable for any damages
+//  arising from the use of this software.
+//
+//  Permission is granted to anyone to use this software for any purpose,
+//  including commercial applications, and to alter it and redistribute it
+//  freely, subject to the following restrictions:
+//
+//  1. The origin of this software must not be misrepresented; you must not
+//     claim that you wrote the original software. If you use this software
+//     in a product, an acknowledgment in the product documentation would be
+//     appreciated but is not required.
+//  2. Altered source versions must be plainly marked as such, and must not be
+//     misrepresented as being the original software.
+//  3. This notice may not be removed or altered from any source distribution.
+
+/*---
+description: Tests the unary '-' operation.
+includes: [simdUtilities.js]
+---*/
+
+simdTypes.filter(isSigned).forEach(function(type) {
+  testSimdFunction(type.name + ' neg', function() {
+    testUnaryOp(type, 'neg', function(a) { return -a; });
+  });
+});
```

Comment:
neg operation is supported for unsigned types too with modulo 2^32 semantics.
See https://github.com/tc39/ecmascript_simd/issues/310. 
It is also part of the spec. See http://tc39.github.io/ecmascript_simd/#simd-neg